doc/DataPlaneCodeGeneration/AzureSDKCodeGeneration_DataPlane_Quickstart.md

```diff
@@ -52,49 +53,102 @@ For this guide, we'll create a getting started project in a branch of your fork
 ```text
 sdk\<service name>\<package name>\README.md
 sdk\<service name>\<package name>\api
+sdk\<service name>\<package name>\perf
+sdk\<service name>\<package name>\samples
 sdk\<service name>\<package name>\src
+sdk\<service name>\<package name>\stress
 sdk\<service name>\<package name>\tests
-sdk\<service name>\<package name>\samples
 sdk\<service name>\<package name>\CHANGELOG.md
+sdk\<service name>\<package name>\<package name>.sln
 ```
 
 - `<service name>` - Should be the short name for the azure service. e.g. deviceupdate
 - `<package name>` -  Should be the name of the shipping package, or an abbreviation that distinguishes the given shipping artifact for the given service. It will be `Azure.<group>.<service>`, e.g. Azure.IoT.DeviceUpdate
+
+### Use Cadl as Input
+
+#### Create Cadl project
   
-We will use dotnet project template [Azure.Template](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/template/Azure.Template) to automatically create the project.
+  ***Initialize Cadl Project***
 
-You can run `eng\scripts\automation\Invoke-DataPlaneGenerateSDKPackage.ps1` to generate the starting SDK client library package directly as following:
+  Follow [Cadl Getting Start](https://github.com/microsoft/cadl/#using-node--npm) to initialize your Cadl project.
 
-```powershell
-eng/scripts/automation/Invoke-DataPlaneGenerateSDKPackage.ps1 -service <servicename> -namespace Azure.<group>.<service> -sdkPath <sdkrepoRootPath> [-inputfiles <inputfilelink>] [-readme <readmeFilelink>] [-securityScope <securityScope>] [-securityHeaderName <securityHeaderName>]
+  Make sure `npx cadl compile .` runs correctly.
+
+  ***Add cadl-csharp emitter***
+
+  Modify package.json, add one line under dependencies:
+
+```diff
+        "dependencies": {
+          "@cadl-lang/compiler": "^0.36.0",
+          "@cadl-lang/rest": "^0.18.0",
+          "@azure-tools/cadl-azure-core": "^0.8.0",
++         "@azure-tools/cadl-csharp": "0.1.6"
```

Comment:
how can we  get to know the latest version of cadl-csharp, shall we ask them to check the latest version in npmjs?

Reply:
It is the template solution. We cannot ask them to apply the latest version since it may not consistent with compiler. Currently in the doc, we specify the workable version.  If future, we may use `latest` instead of actually version.